src/sentry/static/sentry/app/components/events/interfaces/threads/filterThreadInfo.ts

```diff
@@ -0,0 +1,51 @@
+import {Thread, Frame} from 'app/types/events';
```

Comment:
I think we are using .tsx extension everywhere, even if the file does not contain jsx.

Reply:
We have a few other `.ts` files, but generally folks have been using tsx.